src/entry-dev.js

```diff
@@ -1,18 +1 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { BrowserRouter as Router } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import { init } from './store';
-import App from './App';
-import logger from 'redux-logger';
-import getBaseName from './Utilities/getBaseName';
-
-ReactDOM.render(
-    <Provider store={ init(logger).getStore() }>
```

Comment:
@karelhala, 
Tested, works seamlessly! :+1:
Config PR  RedHatInsights/cloud-services-config#460 merged so ready to go. 
One observation:
With current changes from master, the only difference between `entry-dev` & `entry` files is here it passes `logger` to `init` function. With `AppEntry` change, I didn't see `logger` that passed in props for `RosApp` in case of `boostrap-dev`. 
May I add this as props to `RosApp` once these initial changes of chrome2 merged?

Reply:
Sure thing go ahead. You might want to test if `foo` is in the URL and if that's true use the logger.

Reply:
Also,  getting a few service specific warnings. 
Example -
```
WARNING in ./src/Routes.js 86:12-25
Should not import the named export 'routes'.'rosPage' (imported as 'paths') from default-exporting module (only default export is available soon)
 @ ./src/App.js 15:0-34 57:162-168
 @ ./src/AppEntry.js 6:0-24 15:38-41
 @ container entry ./RootApp[0]

```
Do I need to fix these warnings as well?

Reply:
No need to, this is webpack complaining you are importing named export from JSON file. If you want clean console you can import the config as `import config from '../package.json'`. But that's extra and you don't really need to do that.